client/state/data-getters/index.js

```diff
@@ -23,6 +23,11 @@ import {
 import { convertToSnakeCase } from 'state/data-layer/utils';
 import { dummyTaxRate } from 'lib/tax'; // #tax-on-checkout-placeholder
 
+export const getAtURL = url =>
```

Comment:
This could use some inline documentation and/or examples.

Reply:
added in a010ec9